app/controllers/concerns/idv_session.rb

```diff
@@ -33,4 +33,18 @@ def idv_vendor
   def idv_attempter
     @_idv_attempter ||= Idv::Attempter.new(current_user)
   end
+
+  def idv_agent
+    @_agent ||= Proofer::Agent.new(
+      applicant: idv_session.applicant,
+      vendor: (idv_session.vendor || idv_vendor.pick),
+      kbv: FeatureManagement.proofing_requires_kbv?
+    )
+  end
+
+  def init_questions_and_profile(resolution)
+    idv_session.resolution = resolution
+    idv_session.question_number = 0
+    idv_session.profile_from_applicant(idv_session.applicant, password)
+  end
```

Comment:
It's related in the sense that reek complained of too many methods in the controller, so a slight refactor was necessary to get codeclimate passing.